main/models.py

```diff
@@ -133,6 +135,15 @@ def get_full_url(self, proto='https'):
         domain = Site.objects.get_current().domain
         return f'{proto}://{domain}{self.get_absolute_url()}'
 
+    @property
+    def updated_mins_ago(self):
+        now = datetime.utcnow().replace(tzinfo=utc)
+        return (now - self.last_update).seconds // 60
+
+    @property
+    def is_recent(self):
+        return self.updated_mins_ago <= 90
```

Comment:
Is this an educated guess or well informed?

Reply:
I took the mirror sync delays from the [mirror status](https://www.archlinux.org/mirrors/status/), removed all mirrors with more than 6 hours of delay (around 100 of them), then averaged the rest (around 600) - that gives you 1h26min, so I just rounded it up to 90 minutes. This notifies about delays on about 400 mirrors.

Averaging mirrors below 10 hours gives 1h44mins (covers 420 mirrors), below 24 hours gives 3h06mins (covers 500 mirrors) and all mirrors average to about 4 hours (covers 550 mirrors). The longer this period is, the more annoying this notification ends up being for maintainers though, so I took the first average as a good starting point. If people keep complaining too much, it can be always bumped up.